ui/packages/catalog-ui-search/src/main/webapp/react-component/location/polygon.js

```diff
@@ -1,9 +1,10 @@
 const React = require('react');
```

Comment:
❓ Is it worth pulling this and https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/react-component/location/line.js into a base component so they can be reused?  They seem very similar after this update.